src/instructlab/model/evaluate.py

```diff
@@ -314,6 +325,11 @@ def launch_server(
     type=click.Path(),
     help="Path where all the MMLU Branch tasks are stored. Needed for running mmlu_branch.",
 )
+@click.option(
+    "--gpus",
```

Comment:
I think `--num-gpu` might be clearer but I won't block for this.

Reply:
--gpus was used by train.  I was thinking consistency was more important.